### PR TITLE
Adjust listing of submitted advises from integration-tests

### PR DESCRIPTION
**Pull request by fridex**
## This introduces a breaking change

- [x] No

### CODE REVIEW

**Comment by fridex** [comment]
CC @pacospace PTAL if we have all of them.

File: features/thamos_advise.feature [hunk 1]
```diff
@@ -22,6 +22,10 @@ Feature: Running thamos advise against deployment
 
         Examples: Advise
             | git_repo                                                             | runtime_environment     | user_stack  | static_analysis |
-            | https://github.com/thoth-station/elyra-aidevsecops-tutorial          | download_dataset        | without     | without         |
+            | https://github.com/thoth-station/elyra-aidevsecops-tutorial          | download-dataset        | with        | without         |
             | https://github.com/thoth-station/elyra-aidevsecops-tutorial          | training                | without     | with            |
-            | https://github.com/thoth-station/elyra-aidevsecops-tutorial          | test_model              | with        | without         |
+            | https://github.com/thoth-station/elyra-aidevsecops-tutorial          | test_model              | with        | with            |
```
Comment (pacospace) [inline features/thamos_advise.feature:27]: `test-model`, also below

Comment (pacospace) [inline features/thamos_advise.feature:27]: no `static_analysis` is performed yet from jupyterlab-requirements

File: features/thamos_advise.feature [hunk 2]
```diff
@@ -22,6 +22,10 @@ Feature: Running thamos advise against deployment
 
         Examples: Advise
             | git_repo                                                             | runtime_environment     | user_stack  | static_analysis |
-            | https://github.com/thoth-station/elyra-aidevsecops-tutorial          | download_dataset        | without     | without         |
+            | https://github.com/thoth-station/elyra-aidevsecops-tutorial          | download-dataset        | with        | without         |
```
Comment (pacospace) [inline features/thamos_advise.feature:25]: Isn't it `overlays/` missing in all URL to user_stack?

e.g. `https://github.com/thoth-station/elyra-aidevsecops-tutorial/overlays`

File: features/thamos_advise.feature (hunk 1 above)
Comment (fridex) [reply features/thamos_advise.feature:27]: I've added all entries with static analysis turned off to support jupyterlab-requirements. Some are kept turned on to test static analysis submission to the backand.

File: features/thamos_advise.feature (hunk 2 above)
Comment (fridex) [reply features/thamos_advise.feature:25]: The URL is pointing to a Git repo to be cloned. The runtime environment is picked based on the configuration in `.thoth.yaml` respecting `runtime_environment` name stated in the `.thoth.yaml`.

Comment (pacospace) [reply features/thamos_advise.feature:25]: we can remove test-model then, as .thoth.yaml file has only `download-dataset`, `training` and `inference` runtime environments.

**Comment by fridex** [comment]
@pacospace I will merge this to continue with work. Feel free to raise any comments.

**Pull request merged**